source/kernel/src/forth/mod.rs

```diff
@@ -385,7 +388,7 @@ async fn spawn_forth_task(forth: &mut forth3::Forth<MnemosContext>) -> Result<()
         );
         forth3::Error::InternalError
     })?;
-    let host_ctxt = MnemosContext::new(kernel, params, forth.host_ctxt.spawnulator.clone()).await;
+    let host_ctxt = MnemosContext::new(kernel, params).await;
     let child_id = host_ctxt.id;
```

Comment:
I'm not sure where we actually increment this (I didn't look super hard - will do tomorrow), but we might want to use a static atomic for the count (see pr notes about "non hierarchical spawns"), or go ahead and use the maitake task ID instead.